springfox-swagger2/build.gradle

```diff
@@ -39,10 +39,19 @@ dependencies {
   testCompile libs.test
 }
 
+idea {
+
+  module {
+    sourceDirs += project.file('out/production/classes/generated')
+    generatedSourceDirs += project.file('out/production/classes/generated')
+    testSourceDirs += project.file('out/production/classes/generated_tests')
+  }
```

Comment:
We require here.
These settings enable to import classes generated by MapStruct.